test/EasyPost/CarrierAccountTest.php

```diff
@@ -3,6 +3,10 @@
 namespace EasyPost\Test;
 
 use EasyPost\CarrierAccount;
+use EasyPost\EasyPost;
+use EasyPost\FieldError;
+use EasyPost\Test\Fixture;
+use VCR\VCR;
```

Comment:
This isn't using the new `setupCassette` below. When this is done, it won't require any of these.

```suggestion
use EasyPost\Test\Fixture;
```